### ISSUE TITLE
Selecting an item from a Dropdown menu in search mode (non multi) does not work

### ISSUE THREAD

**Issue by rszewczyk** [Semantic-Org/Semantic-UI-React#513]
Can reproduce with `create-react-app myapp && cd myapp && npm i stardust -S && npm start` then change contents of `src/index.js` to:

```
import React from 'react'
import ReactDOM from 'react-dom'
import { Dropdown } from 'stardust'

ReactDOM.render(
  <Dropdown
    search
    selection
    options={[{
      value: 1,
      text: 'One'
    }, {
      value: 2,
      text: 'Two'
    }, {
      value: 3,
      text: 'Three'
    }]}
  />,
  document.getElementById('root')
)
```

Observe that selecting an option by clicking on a menu item does not change the displayed value of the component (selecting via the keyboard does work).

**Comment by levithomason** [comment]
This seems to be working on the doc site.  However, I'll try the steps noted.

**Comment by levithomason** [comment]
I see.  This is an issue for the uncontrolled component.  The selected (highlighted) item is chosen on blur even when the item click is what triggers the blur.

Fix coming, hang tight...

**Comment by levithomason** [comment]
I've added a [doc site example](http://technologyadvice.github.io/stardust/modules/dropdown#uncontrolled), tests, and a fix for this issue.  Released in `stardust@0.44.8`.  Thanks for the report!
